Builder/BuilderInterface.php

```diff
@@ -22,8 +22,11 @@
 interface BuilderInterface
 {
     /**
-     * @param AdminInterface            $admin
-     * @param FieldDescriptionInterface $fieldDescription
+     * Adds missing information to the given field description from the model manager metadata,
```

Comment:
>  I'd say we should put fixFieldDescription on FormContractorInterface and make it private to the "real" builders

I don't understand how you could have a private method in an interface, nor what a "real" builder is, can you elaborate ?